sdk/core/azure-core/src/main/java/com/azure/core/http/policy/RetryPolicyOptions.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.policy;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.http.HttpResponse;
+
+import java.time.temporal.ChronoUnit;
+import java.util.Objects;
+
+import static com.azure.core.util.CoreUtils.isNullOrEmpty;
+
+/**
+ * Configuration options for {@link RetryPolicy}.
+ */
+@Fluent
+public class RetryPolicyOptions {
+
+    private RetryStrategy retryStrategy;
+    private String retryAfterHeader;
+    private ChronoUnit retryAfterTimeUnit;
+
+    /**
+     * Creates a default {@link ExponentialBackoff} for retry policy. It will not use any {@code retryAfterHeader}
```

Comment:
```suggestion
     * Creates a default {@link RetryPolicyOptions} used by a {@link RetryPolicy}. This will use {@link ExponentialBackoff} as the {@link #getRetryStrategy retry strategy} and will ignore retry delay headers.
```